typescript/rules/stylisticIssues.ts

```diff
@@ -1,8 +1,9 @@
 // eslint-disable-next-line import/no-unresolved
-import type { StylisticIssues } from "eslint/rules/stylistic-issues";
+import type { Linter } from "eslint";
```

Comment:
nit: does it still make sense to keep this named stylisticsIssues? I don't think it is hurting anything but it isn't importing that same named module.